cli/service/resources/default-reaction-providers.yaml

```
@@ -66,6 +66,9 @@ spec:
         type: string
       eventGridKey:
         type: string
+      eventGridSchema:
+        type: string
+        default: CloudEvents

```

```suggestion
      eventGridSchema:
        type: string
        enum:
          - "CloudEvents"
          - "EventGrid"
        default: "CloudEvents"
```

This will also enable validation of one of these 2 options.

---

Also, it looks like we are missing the format (packed/unpacked) config value?

---

good catch

---

I think maybe in the future we should make this `format` property a default value in the reaction_provider, so that we dont have to implement this for every reaction